src/mixins/editorMixin.js

```diff
@@ -89,6 +89,10 @@ export default {
 				this.reader?.setSearchQuery(this.searchQuery, newValue)
 			}
 		},
+		'editorContent'() {
```

Comment:
I wonder whether it's really required to watch for changes to `editorContent`. Wouldn't it be enough to watch for `currentPage.id`?